runhouse/servers/env_servlet.py

```diff
@@ -65,6 +62,8 @@ class EnvServlet:
     async def __init__(self, env_name: str, *args, **kwargs):
         self.env_name = env_name
 
+        if "logger" in kwargs.keys() and kwargs.get("logger"):
```

Comment:
instead of passing the actual logger here, wouldn't it be easier to pass the log level, and then adjust accordingly?